docs/topics/db/models.txt

```diff
@@ -861,6 +861,26 @@ code will automatically support those arguments when they are added.
     :data:`~django.db.models.signals.pre_save`, and
     :data:`~django.db.models.signals.post_save` are called.
 
+.. _async-model-methods:
+
+Asynchronous model methods
+==========================
+
+.. versionadded:: 4.2
+
+If you are writing asynchronous views or code, you cannot use the ORM for
+model methods in the way we have described
+above, as you cannot call *blocking* synchronous code from asynchronous
+code - it will block up the event loop (or, more likely, Django will
+notice and raise a ``SynchronousOnlyOperation`` to stop that from
+happening).
+
+Fortunately, you can use some asynchronous model methods that use the database:
+
+- ``asave()`` works asynchronously for :meth:`~Model.save`.
+- ``adelete()`` works asynchronously for :meth:`~Model.delete`.
+- ``arefresh_from_db()`` works asynchronously for :meth:`~Model.refresh_from_db`.
+
```

Comment:
I don't think we need this section, we already have the async topic docs to link to the reference.

Also it is inserted in the wrong place - the below heading “Executing custom SQL” is part of the previous section. The different underline characters `=` and `-` signify different heading levels: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections